src/storage/backend/sqlite/SQLite.cc

```diff
@@ -77,6 +79,11 @@ OperationResult SQLite::DoOpen(OpenResultCallback* cb, RecordValPtr options) {
     sqlite3_enable_shared_cache(1);
 #endif
 
+    // Store whether we're in a cluster. Knowing this means we can simplify other parts of the code.
+    auto local_node_name = id::find_val("Cluster::node")->AsStringVal();
+    if ( local_node_name->Len() > 0 )
+        is_cluster = true;
```

Comment:
If we don't need this one, that'd be great - see other comment about `is_cluster`.